src/controllers/bulk/bulksupported.h

```diff
@@ -1,18 +1,34 @@
+#pragma once
+
+#include <cstdint>
+
 // A list of supported USB bulk devices
 
-#pragma once
+static constexpr std::uint32_t kInvalidInterfaceNumber = 0;
+struct bulk_supported_t {
```

Comment:
Did you consider to make this a class along with the helper functions as members?

Reply:
Well, which helper functions are you referring to exactly? `is_interesting` should belong to the enumerator IMO.

Reply:
Yes. But I have no special demands here.

Reply:
I don't think moving that is an improvement really. So I'd like to keep as is if thats okay for you.

Reply:
Ok